modules/openy_gc_auth/modules/openy_gc_auth_daxko_sso/config/openy_gc_auth.provider.daxko_sso.yml

```diff
@@ -0,0 +1 @@
+redirect_url: '/gated_content'
```

Comment:
Can we use `/gated-content`?

Reply:
That is default link to the gated content node app. that is how it's configured by default in demo content. I've just used it.